File: java/javadpbo/src/com/sem4dpbo/week2/praktikum2/latihanpraktikum2_java/Product.java
package com.sem4dpbo.week2.praktikum2.latihanpraktikum2_java;

public class Product {
    private long price;
    private String idProduct;

//    constructor
    public Product(){}

    public Product(String paramIdProduct, long paramPrice){
        this.price = paramPrice;
        this.idProduct = paramIdProduct;
    }

//    method get and set
    public void setIdProduct(String idProduct) {
        this.idProduct = idProduct;
    }

    public void setPrice(long price) {
        this.price = price;
    }

    public long getPrice() {
        return price;
    }

    public String getIdProduct() {
        return idProduct;
    }
}

File: php/2nd_week/Praktikum2/latihanpraktikum2_php/Product.php
<?php

class Product{
    private $idProduct = "";
    private $price = 0;

    function __construct()
    {
    }

    function setIdProduct($idProduct){
        $this->idProduct = $idProduct;
    }

    function setPrice($price){
        $this->price = $price;
    }

    function getIdProduct(){
        return $this->idProduct;
    }

    function getPrice(){
        return $this->price;
    }

    function __destruct()
    {
    }
}
?>